### PR TITLE
Several minor fixes

### PR DESCRIPTION
**Pull request by mxdanger**
* fixed reference tooltip gradient fading

* fixed language selector arrow
* fixed language settings colors

* fixed commons image structured data names list background

* fixed mobile logout button

* fixed media viewer color, border, buttons, and gradient fading

* fixed preferences border and background
* improved readability of preferences help text

### CODE REVIEW

File: wikipedia-dark.user.css [hunk 1]
```diff
@@ -202,11 +202,14 @@ regexp("https?:\/\/wiki\.(archlinux|mozilla)\.(org|jp)\/.*$") {
...
   }
+  .mw-mmv-info-box {
+    border-color: var(--gray-5) !important;
+  }
   .mw-mmv-image-metadata, .navbox, .navbox-subgroup, .mw-mmv-info-box,
   .mwe-pt-article-row-odd, .mwe-pt-article-row-even, .ve-ui-mwSaveDialog-options,
   #mw-page-base, .mw-parser-output #siteContainer div:first-child,
   div.vectorTabs li.selected {
-    background-color: transparent;
+    background-color: transparent !important;
```
Comment (the-j0k3r) [inline wikipedia-dark.user.css:212]: Please dont add !important to any rules like this, the correct way is to isolate the desired selector and increase its specificity and only by last resort use !important.

File: wikipedia-dark.user.css [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /* ==UserStyle==
 @name         Wikipedia Dark
 @namespace    StylishThemes
-@version      3.4.0
+@version      3.4.1
```
Comment (the-j0k3r) [inline wikipedia-dark.user.css:4]: Please dont include any release commits with PR's,

Thanks.

File: wikipedia-dark.user.css [hunk 3]
```diff
@@ -2998,11 +3011,25 @@ regexp("https?:\/\/wiki\.(archlinux|mozilla)\.(org|jp)\/.*$") {
   .oo-ui-textInputWidget.oo-ui-widget-enabled .oo-ui-inputWidget-input::placeholder {
     color: var(--gray-7) !important;
   }
-  .oo-ui-dropdownWidget.oo-ui-widget-enabled .oo-ui-dropdownWidget-handle,
-  .mw-mmv-label {
+  .mw-mmv-title-para .mw-mmv-ttf-ellipsis {
+    background-color: var(--gray-2);
+  }
+  .oo-ui-dropdownInputWidget.oo-ui-widget-enabled, .oo-ui-dropdownInputWidget.oo-ui-widget-enabled:hover {
+    background-color: transparent !important;
```
Comment (the-j0k3r) [inline wikipedia-dark.user.css:3018]: Avoid `!important` where not absolutely needed, preferably increase specificity/weight of affected selectors

**Comment by the-j0k3r** [comment]
@mxdanger Thank you for reaching out to us with this contribution. =)

Ive made some comments and asked for some minor changes to speed up the process, I would normally test the changes and make more specific suggestions, but Im running a bit hort on time this week.

Thanks in advance for understanding.

**Comment by the-j0k3r** [comment]
@mxdanger please rebase on master after you made the changes and force push =) 

Thanks.

**Comment by mxdanger** [comment]
@the-j0k3r I saw the comments you made. I’ll add those changes to this commit when I have the time to work on it.

File: wikipedia-dark.user.css (hunk 2 above)
Comment (mxdanger) [reply wikipedia-dark.user.css]: My mistake. I've updated it to match master to avoid any conflicts.

File: wikipedia-dark.user.css (hunk 3 above)
Comment (mxdanger) [reply wikipedia-dark.user.css:3020]: I was unable to find a better solution.

File: wikipedia-dark.user.css (hunk 1 above)
Comment (mxdanger) [reply wikipedia-dark.user.css]: I've isolated the selector.

File: wikipedia-dark.user.css [hunk 4]
```diff
@@ -1,7 +1,7 @@
 /* ==UserStyle==
 @name         Wikipedia Dark
 @namespace    StylishThemes
-@version      3.4.1
+@version      3.4.2
```
Comment (the-j0k3r) [inline wikipedia-dark.user.css:4]: Hello, please again, dont include release commits, these are intended for internal use only.

Comment (mxdanger) [reply wikipedia-dark.user.css:4]: Got it. 👍

File: wikipedia-dark.user.css (hunk 2 above)
Comment (the-j0k3r) [reply wikipedia-dark.user.css]: Dont edit this value at all please, master changes and this will screw it up.

**Comment by mxdanger** [comment]
Is there anything left that needs changing before its merged?

**Comment by the-j0k3r** [comment]
> Is there anything left that needs changing before its merged?

I still see a revision commit now at 3.4.2, please rebase this PR on master and ensure that no version commits show up on the diffs.

Until that is resolved I cant merge this.

File: wikipedia-dark.user.css (hunk 3 above)
Comment (the-j0k3r) [reply wikipedia-dark.user.css:3020]: I asked for links to pages where the issue manifested so I could see about better solutions, =(

**Comment by the-j0k3r** [comment]
WIth https://github.com/StylishThemes/Wikipedia-Dark/pull/129/commits/a40c83274adba53d08906530084beb933a26fd0e I resolved all the issues, please test to ensure all works.

**Comment by the-j0k3r** [comment]
Closing in favor of https://github.com/StylishThemes/Wikipedia-Dark/pull/130 @mxdanger please check out and test.

**Pull request closed without merging**